python/langsmith/client.py

```diff
@@ -1444,7 +1458,6 @@ def multipart_ingest(
         update: Optional[
             Sequence[Union[ls_schemas.Run, ls_schemas.RunLikeDict, Dict]]
         ] = None,
```

Comment:
Why remove?

Reply:
can add back if we want this part of public interface. removes some tech debt. basically just keeping this function around for backwards compat with tests to make sure they were working - would have removed otherwise